### ISSUE TITLE
Publish rights

### ISSUE THREAD

**Issue by iskhakovt** [jrief/django-websocket-redis#174]
Is there a way to allow to publish to a channel for specific users?

Currently `publish`/`publish-group`/`publish-user` may be added to url for WebSocket on client side, is there a way to prohibit it?

**Comment by jrief** [comment]
yes, put those users into a group and publish only into that group.

Currently there is no such thing as a channel, ie. kind of temporary group, but it would be quite easy to implement.

**Comment by iskhakovt** [comment]
No, I meant to create a group with different rights -- publisher and subscriber, who is allowed only to receive messages.

**Comment by jrief** [comment]
OK, git it.
This would have to be implemented.
